packages/patternfly-4/react-core/src/components/index.ts

```diff
@@ -17,7 +17,6 @@ export * from './ChipGroup';
 export * from './ClipboardCopy';
 export * from './ContextSelector';
 export * from './DataList';
-export * from './Divider';
```

Comment:
Why did we remove these?  Does that they can only be exported form experimental

Reply:
You will still be able to do
`import { Divider } from '@patternfly/react-core` since the react-core/index barrel file exports out the experimental components which themselves point back into the components folder. For that same reason, Drawer and DataToolbar and not re-exported again in this file.

Reply:
We had to remove Divider, OverflowMenu, and Spinner from /react-core/src/components/index.ts after adding their exports back into Experimental in order to avoid duplicate exports - when we had them in previously it was throwing errors for each and failing the build.  We did this to mirror how Drawer and DataToolbar are currently set up.

Reply:
Ok. I see it now.  thanks.